docs/core/integration-quality-scale/rules/iqs052.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "IQS052 - The documentation provides automation examples the user can use."
+---
+
+IQS052 is as follows:
+
+The documentation provides automation examples the user can use.
```

Comment:
I now changed `examples` -> `automation examples` because if I only remove the whole list of examples, it does not really provide any insights of what `examples` mean